charts/templates/cluster-global-sidecar.yaml

```diff
@@ -238,12 +237,75 @@ spec:
                 key: "Slime-Orig-Dest"
                 value: "%DOWNSTREAM_LOCAL_ADDRESS%"
               append: true
-            - header:
-                key: "Slime-Source-Ns"
-                value: {{ $ns }}
-              append: true
+    - applyTo: HTTP_FILTER
+      match:
+        context: SIDECAR_OUTBOUND
+        listener:
+          name: 0.0.0.0_{{ . }}
```

Comment:
we'd better change svc-port of global sidecar to an uncommon value as the patch will enable lua filter on the whole listener.